tests/e2e/pages/PaymentMethodsPage.js

```diff
@@ -18,4 +20,145 @@ export default class PaymentMethodsPage {
+            .click(oneyDateOfBirth)
+            .typeText(oneyDateOfBirth, shopper.dateOfBirth)
+            .typeText(oneyEmail, shopper.shopperEmail)
+    }
+
+    confirmOneyPayment = async () => {
+        //Simulation on the Oney page
+    }
+
+    initiateKlarnaPayment = async (klarnaVariant) => {
+        let klarnaSelector = Selector('#rb_klarna');
+        if(klarnaVariant){
+            klarnaSelector = klarnaVariant == 'paynow' ? Selector('#rb_klarna_paynow') : Selector('#rb_klarna_account');
+        }
+        await t
+            .click(klarnaSelector)
+    }
+
+    confirmKlarnaPayment = async () => {
+        await Selector('#payment-method-selector')();
+        await t
+            .click(Selector('#buy-button'))
+            .switchToIframe('#klarna-hpp-instance-fullscreen')
+            .typeText(Selector('input[name="dateOfBirth"]'), '01011991')
+            .click(Selector('button').withAttribute('id', /continue/))
+            .switchToMainWindow()
+    }
+
+    cancelKlarnaDirectEBankingPayment = async () => {
+        await t
+            .click(Selector('.back-to-merchant cancel-transaction'))
+            .click(Selector('#CancelTransaction'))
+    }
+
+    cancelKlarnaPayment = async () => {
+        await t
+            .click(Selector('#back-button'))
+    }
+
+    confirmGiropayPayment = async (giroPayData) => {
+        const giroBank = Selector('#ui-id-1 li a');
+        const giroBankDropdown = Selector('#tags');
+        await giroBankDropdown()
+        await t
+            .typeText(giroBankDropdown, giroPayData.bankName)
+        await giroBank()
+        await t
+            .click(giroBank)
+            .click('input[name="continueBtn"]')
+            .click('#yes')
+            .typeText('input[name="sc"]', giroPayData.sc)
+            .typeText('input[name="extensionSc"]', giroPayData.extensionSc)
+            .typeText('input[name="customerName1"]', giroPayData.customerName)
+            .typeText('input[name="customerIBAN"]', giroPayData.customerIban)
+            .click('input[value="Absenden"]')
+    }
+
+    cancelGiropayPayment = async () => {
+        await t
+            .click(Selector('#backUrl'))
+            .click(Selector('.modal-dialog #yes'))
+    }
+
+    initiateEPSPayment = async () => {
+        const epsInput = Selector('input[value="eps"]');
+        const epsDropDown = Selector('#component_eps .adyen-checkout__dropdown__button');
+        const epsIssuer =  Selector('#component_eps .adyen-checkout__dropdown__list li');
+
+        await t
+            .click(Selector('#rb_eps'))
+            .click(epsInput)
+            .click(epsDropDown)
+            .click(epsIssuer)
+    }
+
+    initiateAffirmPayment = async (shopper) => {
+        const affirmEmail = Selector('#component_affirm input[name="shopperEmail"]');
+        await t
+            .click(Selector('#rb_affirm'))
+            .typeText(affirmEmail, shopper.shopperEmail)
+    }
+
+    confirmSimulator = async () => {
```

Comment:
what are these methods for?

Reply:
These methods are for the simulator from SimulatorAcquirerAccount, which is used for multiple payment methods (e.g. EPS). Although, currently this simulator is down, so can't complete the payment. Will have to update it once the simulator works again. Will reach out internally regarding this.